src/FbgemmSpMM.cc

```diff
@@ -0,0 +1,721 @@
+#include "fbgemm/FbgemmSpMM.h"
+
+#include <cpuinfo.h>
+#include <algorithm>
+#include <cassert>
+#include <cstdio>
+#include <cstring>
+#include <memory>
+#include <vector>
+
+#include <fbgemm/Utils.h>
+
+using namespace std;
+
+namespace fbgemm {
+
+namespace x86 = asmjit::x86;
+
+using internal::SpMMTypeTrait;
+
+namespace {
+
+template <class To, class From>
+typename enable_if<
+    (sizeof(To) == sizeof(From)) && is_trivially_copyable<From>::value &&
```

Comment:
FYI `std::is_trivially_copyable` isn't supported by gcc-4's stdlib. I mention this because in README it says FBGEMM needs GCC 4.9+

Reply:
Is this causing a build failure for PyTorch is some configuration?

Reply:
No, PyTorch has dropped gcc-4 support, so this isn't breaking PT